docker/patched-star/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM ubuntu:zesty
+FROM ubuntu:16.04
```

Comment:
@ambrosejcarr This is also a risky change. 

We use `apt install samtools` in this dockerfile, without specifying the version of the samtools, this is dangerous. 

Ubuntu 17.04 provides: samtools Version: 1.3.1 (using htslib 1.3.2) BUT 
Ubuntu 16.04 only provides: samtools Version: 0.1.19-96b5f2294a

we should aware this and think about if there are any scientific packages in other images that are installed by `apt install` but with no version specified. If you don't mind, I'm going to modify this dockerfile to use samtools 1.6, build it from source, like what I did in the samtools dockerfile.

Reply:
just make sure the syntax of samtools are consistent between these two version.